api/encryption-handlers_test.go

```diff
@@ -435,8 +552,37 @@ func (suite *TenantTestSuite) TestTenantEncryptionInfoWithoutError() {
 	suite.assert.Nil(err)
```

Comment:
should we add here an assert of the expected response? cause otherwise we would just be testing that there were no errors. Similar in other related tests that don't return errors.

Reply:
you got it!, added assertion for expected errors and response model.